src/CoreShop/Component/Shipping/Model/ShippableItemInterface.php

```diff
@@ -14,6 +14,18 @@
 
 interface ShippableItemInterface
 {
+    /**
+     * @param bool $withTax
+     * @return int
+     */
+    public function getTotal(bool $withTax = true);
```

Comment:
Is actually a BC break. In Core currency only implemented from `CartItemInterface` so there it is available. Anyone against this break?